app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,103 @@
  *= require_tree .
  *= require_self
  */
+html {
+    background-color: #343535b0;
+}
+
+.body-text {
+    background-color: rgb(209, 207, 207);
+    padding-bottom: 20px;
+    padding-top: 20px;
+}
+
+li {
+    list-style-type: none;
+    margin: 0 2vw;
+    font-size: 2vh;
+}
+
+nav {
+    width: 100vw;
+    background-color: #D6EFFF;
+}
+
+.header {
+    margin-left: 20px;
+    width: 100%;
+    text-decoration: none;
+    color: white;
+    padding: 2vw;
+    font-family: 'Courier New', Courier, monospace;
+}
+
+.link {
+    text-decoration: none;
+    color: black;
+    padding: 1vw;
+    font-family: 'Courier New', Courier, monospace;
+}
+
+ul {
+    margin: 0;
+    padding-top: 6px;
+    height: 40px;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    top: 50%;
+}
+
+a {
+    color: rgb(15, 15, 15);
+    margin-left: 10px;
+    text-decoration: none;
+}
+
+a:hover {
+    text-decoration: underline;
+}
+
+p {
+    margin-left: 5px;
+    color: black
+}
+
+h2, h3, form, button{
+    margin-left: 5px;
+    padding: 0;
+}
+
+input {
+    width: auto;
+    height: auto;
+    margin-top: 7px;
+    margin-bottom: 10px;
+}
+
+input#post_title {
+    width: 20%;
+    height: 30px;
+}
+
+input#post_content {
+    width: 60%;
+    height: 70px;
+    justify-content: start;
+}
+  
+body {
+    margin: 0;
+    font-family: 'Courier New', Courier, monospace;
+    color: rgb(19, 19, 19);
+}
+
+.notice {
+    text-align: center;
+    width: 99%;
+    padding: 10px 0 10px 0; 
```

Comment:
A nice little shortcut for this would be:

```suggestion
    padding: 10px 0; 
```

First value is for top & bottom and second is for left and right.

Reply:
Pretty handy tbf... If I'm honest I just smashed the CSS together as quick as I could but I really appreciate the tips and advice as I've really only used CSS in code that only I would be looking at myself  :)

Reply:
Yep, these are all just advisory type comments to give you some tips & info really.